hydrolib/core/io/crosssection/models.py

```diff
@@ -139,9 +139,10 @@ def validate_friction_specification(cls, values):
 
             if frictionid == "":
                 if frictiontype == "" or frictionvalue == "":
-                    raise ValueError(
-                        f"Cross section is missing any friction specification."
-                    )
+                    # No validation error: frictionid will always default to some
```

Comment:
So if the kernel is not provided a friction it calculate with the default?

Reply:
indeed, the frictionId "Main", etc., by the kernel. Intentionally leaving it blank on the Python / serialization side.